synapse/models/transport.py

```diff
@@ -52,17 +106,177 @@ def getModelDefs(self):
+                            'doc': 'The actual departure {point}.'}),
+
+                        ('arrived', ('time', {}), {
+                            'doc': 'The actual arrival time.'}),
+
+                        ('arrived:place', ('geo:place', {}), {
+                            'doc': 'The actual arrival {place}.'}),
+
+                        ('arrived:point', ('transport:point', {}), {
+                            'doc': 'The actual arrival {point}.'}),
+
+                        ('scheduled:duration', ('duration', {}), {
+                            'doc': 'The scheduled duration.'}),
+
+                        ('scheduled:departure', ('time', {}), {
+                            'doc': 'The scheduled departure time.'}),
+
+                        ('scheduled:departure:place', ('geo:place', {}), {
+                            'doc': 'The scheduled departure {place}.'}),
+
+                        ('scheduled:departure:point', ('transport:point', {}), {
+                            'doc': 'The scheduled departure {point}.'}),
+
+                        ('scheduled:arrival', ('time', {}), {
+                            'doc': 'The scheduled arrival time.'}),
+
+                        ('scheduled:arrival:place', ('geo:place', {}), {
+                            'doc': 'The scheduled departure {place}.'}),
+
+                        ('scheduled:arrival:point', ('transport:point', {}), {
+                            'doc': 'The scheduled arrival {point}.'}),
+                    ),
+                }),
+
+                ('transport:trip', {
+                    # train, flight, drive, launch...
+                    'doc': 'Properties common to a specific trip taken by a vehicle.',
+                    'interfaces': ('transport:schedule',),
+
+                    'props': (
+
+                        ('status', ('str', {'enums': 'scheduled,cancelled,in-progress,completed,aborted,failed,unknown'}), {
+                            'doc': 'The status of the {trip}.'}),
+
+                        ('occupants', ('int', {'min': 0}), {
+                            'doc': 'The number of occupants of the {vehicle} on this {trip}.'}),
+
+                        ('cargo:mass', ('mass', {}), {
+                            'doc': 'The cargo mass carried by the {vehicle} on this {trip}.'}),
+
+                        ('cargo:volume', ('geo:dist', {}), {
+                            'doc': 'The cargo volume carried by the {vehicle} on this {trip}.'}),
+
+                        ('operator', ('ps:contact', {}), {
+                            'doc': 'The contact information of the operator of the {trip}.'}),
+
+                        ('vehicle', ('transport:vehicle', {}), {
+                            'doc': 'The {vehicle} which traveled the {trip}.'}),
+                    ),
+                }),
+            ),
+            'edges': (
             ),
             'forms': (
+
+                ('transport:stop', {}, (
```

Comment:
Does the time/duration of the stop get captured by .seen?

Reply:
`transport:schedule:duration` is inherited.